hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -442,22 +442,31 @@
       uvm_test_seq: chip_sw_all_escalation_resets_vseq
       sw_images: ["//sw/device/tests/sim_dv:all_escalation_resets_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
-      run_opts: ["+bypass_alert_ready_to_end_check=1"]
+      run_opts: [
+        "+do_creator_sw_cfg_ast_cfg=1",
```

Comment:
Are you making this change for all tests?

Reply:
I suppose we should need to set `do_creator_sw_cfg_ast_cfg=1` for tests that don't run rom_start.S, or have an OTP image that sets ast_init_en incorrectly. So it seems strange we need to set this plusargs for so many tests.

Reply:
I am not 100% sure, but I think @moidx just added it for all tests since they implicitly had it set before.
We probably still need to go through the exercise of removing the plusarg from tests that do not strictly require it.

Reply:
Two points:
- Please avoid adding "+do_creator_sw_cfg_ast_cfg=1" for each test, and simply add it to the list at line 229, which will affect all tests.
- Regardless of that, can we simply avoid setting that in the PR, run a full regression on the PR to identify the failures, and set the flag for the failures only? That way we will make progress.